src/views/Nft/market/hooks/useGetLowestPrice.ts

```diff
@@ -0,0 +1,73 @@
+import { useEffect, useState } from 'react'
+import { getNftsMarketData } from 'state/nftMarket/helpers'
+import { NftToken } from 'state/nftMarket/types'
+import { pancakeBunniesAddress } from '../constants'
+
+export interface LowestNftPrice {
+  isFetching: boolean
+  lowestPrice: number
+}
+
+export const getBunnyIdFromNft = (nft: NftToken): string =>
+  nft.attributes?.find((attr) => attr.traitType === 'bunnyId').value.toString()
+
+export const useGetLowestPriceFromBunnyId = (bunnyId: string): LowestNftPrice => {
+  const [isFetching, setIsFetching] = useState<boolean>(false)
+  const [lowestPrice, setLowestPrice] = useState<number>(null)
+
+  useEffect(() => {
+    const fetchLowestPrice = async () => {
+      try {
+        setIsFetching(true)
+        const response = await getNftsMarketData({ otherId: bunnyId, isTradable: true }, 1, 'currentAskPrice', 'asc')
+
+        if (response.length > 0) {
+          const [tokenMarketData] = response
+          setLowestPrice(parseFloat(tokenMarketData.currentAskPrice))
+        }
+      } finally {
+        setIsFetching(false)
+      }
+    }
+
+    if (bunnyId) {
+      fetchLowestPrice()
+    }
+  }, [bunnyId])
+
+  return { isFetching, lowestPrice }
+}
+
+export const useGetLowestPriceFromNft = (nft: NftToken): LowestNftPrice => {
+  const [isFetching, setIsFetching] = useState<boolean>(false)
+  const [lowestPrice, setLowestPrice] = useState<number>(null)
+  const isPancakeBunny = nft.collectionAddress?.toLowerCase() === pancakeBunniesAddress.toLowerCase()
+
+  useEffect(() => {
+    const fetchLowestPrice = async () => {
+      const bunnyIdAttr = getBunnyIdFromNft(nft)
```

Comment:
Maybe
- Take `const bunnyIdAttr = getBunnyIdFromNft(nft)` can be taken out of `useEffect` 
- Call `useGetLowestPriceFromBunnyId(bunnyIdAttr)`
- Remove useStates (as they are in the other hook already)

i.e.
```
export const useGetLowestPriceFromNft = (nft: NftToken): LowestNftPrice => {
   const isPancakeBunny = nft.collectionAddress?.toLowerCase() === pancakeBunniesAddress.toLowerCase()
   const bunnyIdAttr = getBunnyIdFromNft(nft) ?? null
   const { isFetching, lowestPrice } = useGetLowestPriceFromBunnyId(bunnyIdAttr)
   if (isPancakeBunny) {
    return { isFetching, lowestPrice }
   }
  return 0 // to be done later for other collections
}
```

Reply:
agh, for other collectibles it should `return { isFetching: false, lowestPrice: nullOrZero }` to be compatible with some components. Its not gonna hit anything yet though, but as placeholder it better replicate what we gonna have in the future.

Reply:
Yea I see what you mean - I'd rather not couple the hooks at this point though. This logic may well change - particularly as we develop the lowest priced traits UI & subgraph queries.

In the immediate term though, we need a way to load the `CollectibleCard`s without the app crashing.

Reply:
No problem 👍 
Yeah it probably will change a lot